chainstate/storage/src/internal/mod.rs

```diff
@@ -151,6 +152,7 @@ impl<B: storage::Backend> BlockchainStorageRead for Store<B> {
     delegate_to_transaction! {
         fn get_storage_version(&self) -> crate::Result<u32>;
         fn get_best_block_id(&self) -> crate::Result<Option<Id<GenBlock>>>;
+        fn get_genesis_block_id(&self) -> crate::Result<Option<Id<Genesis>>>;
```

Comment:
I don't think this is necessary. I'll write in the relevant place how to avoid this.